mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+      analyses. The HADISST1 spanned 1870 onward but the OI.v2, which started
+      in November 1981, better resolved features such as the Gulf Stream and
+      Kuroshio Current which are important components of the climate system.
+      Since the two data sets used different development methods, anomalies
+      from a base period were used to create a more homogeneous record. Also,
+      additional adjustments were made to the SIC data set.
+    </description>
+    <source>
+      [NCAR Hadley-NOAA/OI SST website]
+      (https://climatedataguide.ucar.edu/climate-data/merged-hadley-noaaoi-sea-surface-temperature-sea-ice-concentration-hurrell-et-al-2008)
+    </source>
+    <releasePolicy>
+      Unknown
+    </releasePolicy>
+    <references>
+      [Hurrell et al. (2008)](https://doi.org/10.1175/2008JCLI2292.1)
+    </references>
+    <bibtex>
+      @article{Hurrell2008,
+      author = {James W.  Hurrell  and  James J.  Hack  and  Dennis  Shea  and  Julie M.  Caron  and  James  Rosinski},
+      title = {A New Sea Surface Temperature and Sea Ice Boundary Dataset for the Community Atmosphere Model},
+      journal = {Journal of Climate},
+      volume = {21},
+      number = {19},
+      pages = {5145-5153},
+      year = {2008},
+      doi = {10.1175/2008JCLI2292.1},
+      URL = {https://doi.org/10.1175/2008JCLI2292.1}
+      }
+    </bibtex>
+    <dataUrls>
+      - ftp://ftp.cgd.ucar.edu/archive/SSTICE/MODEL.SST.HAD187001-198110.OI198111-201712.nc
+    </dataUrls>
+    <preprocessing>
+    </preprocessing>
+    <tasks>
+      - climatologyMapSST
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      SSS from NASA Aquarius satellite
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+      Level 3 Aquarius sea surface salinity (SSS) data products have a temporal
+      resolutions of daily, 8 day, monthly, 3 months, and annual.  Monthly and
+      seasonal climatology
+      products from Aqaurius are also available. The Aquarius instrument
+      provides global coverage every 7 days. L3 products are gridded at 1
+      degree spatial resolution.
+    </description>
+    <source>
+      [NASA Aquarius FTP server]
+      (ftp://podaac-ftp.jpl.nasa.gov/allData/aquarius/L3/mapped/V4/7day_running/SCI/)
+    </source>
+    <releasePolicy>
+      NASA data are not copyrighted; however, when you publish our data or
+      results derived therefrom, we request that you include an acknowledgment
+      within the text of the publication and reference list.
+      [Data Citation and Acknowledgements]
+      (https://podaac.jpl.nasa.gov/CitingPODAAC)
+    </releasePolicy>
+    <references>
+      [Lagerloef et al. (2015)](ftp://podaac.jpl.nasa.gov/SalinityDensity/aquarius/docs/v4/AQ-014-PS-0016_AquariusSalinityDataValidationAnalysis_DatasetVersion4.0and3.0.pdf)
+    </references>
+    <bibtex>
+    </bibtex>
+    <dataUrls>
+      (missing)
+    </dataUrls>
+    <preprocessing>
+      (missing)
```

Comment:
@milenaveneziani, I believe there is no preprocessing, correct?

Reply:
left preprocessing as 'missing', but I added the source link that points to the blended/OI product.